mitmproxy/tools/console/consoleaddons.py

```diff
@@ -378,6 +379,9 @@ def edit_focus(self, part: str) -> None:
         # but for now it is.
         if not flow:
             raise exceptions.CommandError("No flow selected.")
+        if part in ("response-headers", "response-body", "set-cookies"):
+            if flow.response is None:
+                flow.response = http.HTTPResponse.make()
```

Comment:
To make the code a bit more precise and clearer to read: please combine the two conditions into a single if.

Reply:
@Kriechi I thought about this, but one line will be more, than 79 symbols. Is it allowed to break this rule in Mitmproxy sometimes?

Reply:
hehe - true 😄 
You can add a line break with `\`, or extract all condition into a bool variable, similar to https://github.com/mitmproxy/mitmproxy/blob/master/mitmproxy/proxy/root_context.py#L109-L114.